modules/kmm-using-tolerations-for-kernel-module-scheduling.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-using-tolerations-for-kernel-module-scheduling_{context}"]
+= Using tolerations for kernel module scheduling
+
+There are circumstances where you will need to evacuate workloads on a node before upgrading a kernel module, which you can do through taints. 
+A taint allows a node to refuse a pod to be scheduled unless that pod has a matching toleration.
```

Comment:
```suggestion
You can use a taint to schedule only pods that contain a matching toleration on the node.
```

 
Avoid using "allow" for inanimate objects: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.

Reply:
Made this change.